tests/php/Control/UserDefinedFormControllerTest.php

```diff
@@ -128,11 +129,17 @@ public function testProcess()
         $this->assertStringEndsWith('#uff', $location);
 
         // check that multiple email addresses are supported in to and from
-        $this->assertEmailSent(
-            'test1@example.com; test2@example.com',
-            'test3@example.com; test4@example.com',
-            'Test Email'
-        );
+        // space between addresses is for framework 4.8 and lower which does not trim() email addresses
+        // no space addresses is for framework 4.9 which does trim() email addresses
+        $to = 'test1@example.com; test2@example.com';
+        $from = 'test3@example.com; test4@example.com';
+        $found = (bool)static::findEmail($to, $from, 'Test Email');
```

Comment:
why not adjust `findEmail()` rather than adding this condition

Reply:
Good idea, I'll do that and close this PR